src/controllers/user/routes.ts

```diff
@@ -7,9 +7,13 @@ const userRouter: Router = Router();
 
 userRouter.route('/')
 .get(authMiddleware('getUser', 'read'), validationHandler( validation.get), UserController.get)
-.post(authMiddleware('getUser', 'write'), validationHandler( validation.create ), UserController.create)
 .put(authMiddleware('getUser', 'write'), validationHandler(validation.update ), UserController.update);
 
+userRouter.get('/getall', authMiddleware('getUser', 'all'), validationHandler(validation.get), UserController.getAll);
```

Comment:
No need to create a new GET route "user/getAll" use simple GET "user/" to get all records, And permission required for this route should be "get" not "all", "all" will restrict this route to "head-trainer" only.  So use `authMiddleware('getUser', 'get')`.